0006/README.md

```diff
@@ -206,6 +206,31 @@ Git branch is the way to manage versioned specifications.
     `/docs/github-actions-workflow/<version>.md`. All specification files in the
     same branch share the same version.
 
+#### Code Contribution
+
+Since the repo uses the Git submodule in a non-standard way and for easy
+building website purposes, people SHOULD NOT use submodules to commit any
+change.
+
+When a community member needs to make a contribution, they fork the repo (and
+clone it locally if they are familiar with git). If the change is for a
+versioned file (a.k.a. spec file), they should make the change in the version
+branch, and send a PR to the version branch. Meanwhile, if the change is for an
+unversioned file or the website itself, they should make the change in `main`,
+and send a PR to `main`.
```

Comment:
I don't think this is workable as-is. If you check out a release branch, you can't preview the changes with Jekyll since it just contains Markdown but not the full site. To preview you need to check out `main` and update the submodule. But if you edit there, it's really hard to commit it as a branch.

Upon second thought, maybe submodules aren't a great fit here. Let me try something out...

Reply:
I agree and think maybe people are operated very differently from I am. I tend to deal with one small thing at a time. When I think about making spec changes, regardless of big or small, that is really a markdown and it has already had a markdown preview in github.com. Including this spec change in the website + preview the website is another change, and it feel like a responsibility from the repo owner.

(And yes, thanks Mark for helping with other alternatives!)